QUEST_LV_0100.tsv

```diff
@@ -1,124 +1,124 @@
+QUEST_LV_0100_20150317_000031	$Take it. I made some bait from old grains in the supplies.{nl}I will lure it from up there, so you stay here.{nl}Sparnas will definitely appear at one of these two places. Don't let it escape. Destroy it for sure.{nl}
+QUEST_LV_0100_20150317_000032	$We may not get a second chance if we don't defeat it this time.{nl}It doesn't seem like they're idiots.
+QUEST_LV_0100_20150317_000033	{memo X}$I heard the sound of fighting so I came in a hurry. Did you defeat it?{nl}I only spotted Shaulems at my spot.{nl}Thank you. You have performed a meritorious deed.
+QUEST_LV_0100_20150317_000034	$Adjutant Paulina
+QUEST_LV_0100_20150317_000035	$We should recover the stolen supplies.{nl}We can't call ourselves soldiers of the Kingdom if we don't do anything about it.{nl}
+QUEST_LV_0100_20150317_000036	xx$The stolen supplies were taken by the Shaulems in every Supply Warehouse, recover the supplies and bring them to Volda.
+QUEST_LV_0100_20150317_000037	$I feel so fed up having to endure this all the time.{nl}Reinforcements should be coming soon, but what can we expect from such an incompetent kingdom?
+QUEST_LV_0100_20150317_000038	$Adjutant Volda
+QUEST_LV_0100_20150317_000039	$Thank you very much, but you must be late for other things.{nl}Besides we can't just sit here and wait. We plan to keep attacking.
+QUEST_LV_0100_20150317_000040	$Secret Guardian
+QUEST_LV_0100_20150317_000041	$The Goddess already foresaw this.{nl}Nothing can escape from her vision.
+QUEST_LV_0100_20150317_000042	$Liaison Officer Niels
+QUEST_LV_0100_20150317_000043	$Hello.{nl}Have you seen an old man who lost his way?{nl}His name is Jonas.{nl}
+QUEST_LV_0100_20150317_000044	$You haven't seen him?{nl}Now which way would that feeble old fellow have gone to?
+QUEST_LV_0100_20150317_000045	$Jonas is a descendant of the family that has been protecting this Mausoleum.{nl}He is a very important person to our lab team so please take good care of him.
+QUEST_LV_0100_20150317_000046	$Archivist Jonas
+QUEST_LV_0100_20150317_000047	$Our family has been waiting for you for a long time.{nl}It is time to give you something that we have been protecting for a long time.
+QUEST_LV_0100_20150317_000048	$It is good to hear that Jonas has regained his consciousness.{nl}You are the person who lifted the light signal, right?{nl}
+QUEST_LV_0100_20150317_000049	$Ahylas, who saw the light, has arrived.{nl}Meet him downstairs.
+QUEST_LV_0100_20150317_000050	$We haven't heard any news from Kepeck for a long time after we sent him supplies.{nl}We don't have enough people to form a search team, so please help us find him.
+QUEST_LV_0100_20150317_000051	$Historian Beard
+QUEST_LV_0100_20150317_000052	$Always be prepared for monster attacks.{nl}Especially in places like here.
+QUEST_LV_0100_20150317_000053	$Mercenary Mirta
+QUEST_LV_0100_20150317_000054	{memo X}$Didn't I already warn you?{nl}Please be careful next time.
```

Comment:
Well with these {memo X} insertions, shouldn't it not matter much because they can search for tab+$ or {memo X}+$?

In any case, I'll put them in the front (which can be searched as tab+$ or $+{memo X})